src/electron/electron/state.cljs

```diff
@@ -5,11 +5,15 @@
 (defonce persistent-dbs-chan (async/chan 1))
 
 (defonce state
-  (atom {:graph/current nil
-         :git/auto-commit-interval nil
+  (atom {:git/auto-commit-interval nil
 
          :config (config/get-config)
-         :db/persisted-before-closing-non-main-window? nil}))
+
+         ;; FIXME: replace with :window/graph
+         :graph/current nil
+
+         ;; window id -> current graph
```

Comment:
it's actually a BrowserWindow object

Reply:
And its k-v is never deleted when a window closes.

Reply:
Good catch! Both are fixed now.